### ISSUE TITLE
HPWH Improvements

### ISSUE THREAD

**Issue by jmaguire1** [NREL/OpenStudio-HPXML#1170]
Supersedes #932 and #862

Here's the high level TODO list for HPWH modeling, in order of priority (from my perspective):

1. Code cleanup: remove any unnecessary EMS programs, any other simplifications we can make?
2. Split systems (including CO2). This is getting worked on as part of a standards funded project, but it's likely that at the end of that project we might need to do a little more validation/testing and add new HPXML extension elements to properly wrap this up.
3. 120 V HPWHs: Main issue here is lack of data, but when we get data incorporating these appliances is high priority for any electrification study.
4. Confined space impacts: Ben Larson did a ton of additional research on this, waiting to pick his brain on if we ought to change our approach to confined space modeling. This one will require an EMS program.
5. Ducting: On hold until we use AFN in OS-HPXML, we'll revisit then. 
6. Combi applications: Will be used during ICF gas HPWH work, currently no E+ limitations we're aware of.
7. Multifunction devices (single HP for water heating, space heating, and cooling): Not widely available yet, and we'd need a good amount of information about how exactly the system works before we can do much evaluation of whether E+ is up to it or not. Put on hold until either these gain more market adoption or more users request this.
